docs/api/js/getting-started.md

```diff
@@ -3,6 +3,12 @@
 <TocHeader />
 <TOC class="table-of-contents" :include-level="[2,3]" />
 
+::: warning
+
+Please note that in the new `0.13.0` SDK the API layer operates with gas. The ergs concept is used by VM only.
+
+:::
```

Comment:
::: warning

Please note, that in the new `0.13.0` SDK version, the API/Node layer operates with gas. The ergs concept is used by VM only.

:::

@uF4No  what we rephrase it like this?

Reply:
Done.